cmd/dmsg-server/commands/root.go

```diff
@@ -53,6 +59,22 @@ var rootCmd = &cobra.Command{
 		if err := sf.ParseConfig(os.Args, true, &conf); err != nil {
 			log.WithError(err).Fatal()
 		}
+		if sentryDSN != "" {
+			err := sentry.Init(sentry.ClientOptions{
+				Dsn: sentryDSN,
+				// Either set environment and release here or set the SENTRY_ENVIRONMENT
+				// and SENTRY_RELEASE environment variables.
+				Environment: "",
+				Release:     "",
+				// Enable printing of SDK debug messages.
+				// Useful when getting started or trying to figure something out.
+				Debug: true,
+			})
+			if err != nil {
+				log.Fatalf("sentry.Init: %s", err)
+			}
+			defer sentry.Flush(2 * time.Second)
```

Comment:
Why is this value equal to 2 seconds?

Reply:
I copied it from the official documentation. I think they do not allow to allow requests too often and have implemented bufferasation of events.